kickthemout.py

```diff
@@ -737,6 +739,17 @@ def targetList(option, opt, value, parser):
         callback=targetList, type='string',
         dest='targets', help='specify target IP address(es) and perform attack')
 
+
+    def target_files(option, opt, value, parser):
+        with open(value) as file:
+            addr_list = file.readlines()
+            setattr(parser.values, option.dest, addr_list)
+
+    parser.add_option('-f', '--file', action='callback',
+        callback=target_files, type='string',metavar='FILE',
+        dest='targets', help='read target IP address(es), one per line, from given file and perform attack')
```

Comment:
- I believe `read target IP address(es) from line-separated file and perform attack` sounds better